### PR TITLE
[feature][ebos] Adding oil-water-polymer option of ebos.

### PR DESCRIPTION
**Pull request by dr-robertk**
This PR simply adds an oil-water-polymer option for the ebos simulator.

### CODE REVIEW

**Comment by dr-robertk** [comment]
jenkins build this please

**Comment by dr-robertk** [comment]
@totto82: You could look at this if you have time.

File: ebos/mebos_main.cc [hunk 1]
```diff
@@ -119,6 +120,23 @@ int main(int argc, char **argv)
...
+        }
+
+        if (polymerActive) {
+            notSupportedErrorStream << "\n"
+                                    << "combining twophase (not water and oil) and polymer is not supported by the multiplexed simulator\n";
+            std::cerr << notSupportedErrorStream.str() << std::endl;
+            std::abort();
+        }
+
         if (polymerActive) {
```
Comment (totto82) [inline ebos/mebos_main.cc]: Why did you keep the line below?

**Comment by dr-robertk** [comment]
jenkins build this please

**Comment by akva2** [comment]
jenkins build this please

File: ebos/ebos_oilwaterpolymer.cc [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    Vanguard::setExternalParseContext(parseContext);
+    Vanguard::setExternalErrorGuard(errorGuard);
+    Vanguard::setExternalDeck(deck);
+}
+
+int ebosOilWaterPolymerMain(int argc, char **argv)
+{
+    typedef TTAG(EbosOilWaterPolymerTypeTag) ProblemTypeTag;
+    return Opm::start<ProblemTypeTag>(argc, argv);
+}
```
Comment (totto82) [inline ebos/ebos_oilwaterpolymer.cc]: This should be startEbos if we want the logging.

Comment (dr-robertk) [reply ebos/ebos_oilwaterpolymer.cc]: I changed it accordingly.

**Comment by dr-robertk** [comment]
jenkins build this please

**Comment by dr-robertk** [comment]
@totto82: Ready from my side.

**Comment by totto82** [comment]
Thanks merging.

**Pull request merged**